ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-12.yaml

```
@@ -46,6 +46,17 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-gcp
   timeout: 5h0m0s
+- as: core-bpf-qa-e2e-tests
+  optional: true
+  run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|COLLECTOR_VERSION|EXPECTED_GO_VERSION)
```

Nightlies are all run as postsubmit (this change will also remove the added nightlies-presubmit file.
```suggestion
  postsubmit: true
```